chia/consensus/blockchain.py

```diff
@@ -637,7 +637,9 @@ def clean_block_records(self):
     async def get_block_records_in_range(self, start: int, stop: int) -> Dict[bytes32, BlockRecord]:
         return await self.block_store.get_block_records_in_range(start, stop)
 
-    async def get_header_blocks_in_range(self, start: int, stop: int) -> Dict[bytes32, HeaderBlock]:
+    async def get_header_blocks_in_range(
+        self, start: int, stop: int, no_tx_filter: bool = False
```

Comment:
if this was C++, I would strongly suggest turning that `bool` into some kind of flags parameter, to make call sites clear on what that `True` or `False` really mean. However, in python you can just name the parameter. Is there a way to force callers to always name the parameter? (if so, I think that would be appropriate). In either case, I think it would be good to always name this parameter when calling the function. A reader will have a hard time understanding what a `True` or `False` means otherwise.

This comment applies to all calls that now take this parameter and passes it on.

Another minor point is that it might be slightly simpler to read this to remove the `no_` prefix, and invert the meaning. Otherwise you end up with a double-negation when following this code.